data/data-pipeline/data_pipeline/etl/sources/census_acs/etl_utils.py

```diff
@@ -22,32 +24,34 @@ def retrieve_census_acs_data(
     tract_output_field_name: str,
     data_path_for_fips_codes: Path,
     acs_type="acs5",
-    raise_errors: bool = False,
 ) -> pd.DataFrame:
     """Retrieves and combines census ACS data for a given year."""
     dfs = []
     for fips in get_state_fips_codes(data_path_for_fips_codes):
-        logger.info(
-            f"Downloading data for state/territory with FIPS code {fips}"
-        )
-
-        try:
-            response = censusdata.download(
-                src=acs_type,
-                year=acs_year,
-                geo=censusdata.censusgeo(
-                    [("state", fips), ("county", "*"), ("tract", "*")]
-                ),
-                var=variables,
+        if fips in CENSUS_ACS_FIPS_CODES_TO_SKIP:
+            logger.info(
+                f"Skipping download for state/territory with FIPS code {fips}"
             )
-            dfs.append(response)
-
-        except ValueError as e:
-            logger.error(
-                f"Could not download data for state/territory with FIPS code {fips}"
+        else:
+            logger.info(
+                f"Downloading data for state/territory with FIPS code {fips}"
             )
 
-            if raise_errors:
+            try:
+                response = censusdata.download(
+                    src=acs_type,
+                    year=acs_year,
+                    geo=censusdata.censusgeo(
+                        [("state", fips), ("county", "*"), ("tract", "*")]
+                    ),
+                    var=variables,
+                )
+                dfs.append(response)
+
+            except ValueError as e:
```

Comment:
@lucasmbrown-usds I was thinking about the discrepancy in the number of census tracts. Do you think we should capture the number of unique tracts that are accessible (for accounting purposes)?